hack/e2e.go

```diff
@@ -563,6 +568,9 @@ func KubemarkTest() error {
 	if err != nil {
 		return err
 	}
+	// If we tried to bring the Kubemark cluster up, make a courtesy
+	// attempt to bring it down so we're not leaving resources around.
+	defer finishRunning("Stop kubemark", exec.Command("./test/kubemark/stop-kubemark.sh"))
```

Comment:
Yes it is - kubemark creates one additional VM that is not a part of the original k8s cluster. It can be done in any order though.